components/feature/downloads/src/test/java/mozilla/components/feature/downloads/manager/AndroidDownloadManagerTest.kt

```diff
@@ -120,7 +120,7 @@ class AndroidDownloadManagerTest {
     }
 
     @Test
-    fun `sendBroadcast with completed download removes queued download from store`() {
+    fun `sendBroadcast with completed download does not remove queued download from store`() {
```

Comment:
nit: As now we don't need to remove the downloads, we could remove the `does not remove queued download from the store` check,  this function can be named as `sendBroadcast with completed download`